### DISCUSSION TITLE
Is this good on Android Tv?

### DISCUSSION THREAD

**Discussion by Internet2022** [aniyomiorg/aniyomi#841] [Q&A]
Is this good on Android Tv?

**Comment by LordZeek** [comment]
Its not perfect and if youve never used the app before it takes a litle setup work but over all Im really happy with it. One issue I had was my controller/remote would not work while using the default player. To remedy this is installed Mx player and use it instead.

**Reply by MikeyKempachi** [reply]
Gostaria de poder também de usar na Smart TV.
Tentei instalar e veio uma lista de bugs para reportar.🧐
